src/file_readers/common/sql_reader.rs

```diff
@@ -12,21 +12,27 @@ pub struct SqlReader {
 }
 
 impl SqlReader {
-    fn read_column_from_table<
-        T: rusqlite::types::FromSql + std::default::Default,
-    >(
+    fn read_column_from_table<T: rusqlite::types::FromSql + Default>(
         &self,
         column_name: &str,
         table_name: &str,
     ) -> Vec<T> {
-        let connection: Connection = get_sql_connection(&self.path);
         let column_names: Vec<String> =
             self.get_table_columns(table_name).unwrap();
         let order_by: String = column_names.join(", ");
         let query: String = format!(
             "SELECT {} FROM {} ORDER BY {}",
             column_name, table_name, order_by
         );
+
```

Comment:
I like that you adhere to the boy scout rule here and leave things better than you found them. Good addition to improve code readability, thanks!